docs/install/Anthos.md

```diff
@@ -19,15 +19,15 @@ This guide will walk you through installing Ondat onto an [Anthos cluster](https
 
 > ⚠️ Make sure to add an [Ondat licence](/docs/operations/licensing/) after installing.
 
-> ⚠️ Make sure you have a running Anthos user cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
+> ⚠️ Make sure you have a running Google Anthos cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
 
-> ⚠️ Make sure your Anthos clusters use [ubuntu_containerd](https://cloud.google.com/anthos/clusters/docs/on-prem/1.8/concepts/using-containerd) as the default node operating system. This version was chosen as the image includes the native Linux components required for Ondat.
+> ⚠️ Make sure your Google Anthos cluster use [`ubuntu_containerd`](https://cloud.google.com/anthos/clusters/docs/on-prem/1.8/concepts/using-containerd) as the default node operating system. This node operating system image has the required kernel modules available for Ondat to run successfully.
```

Comment:
would we need to change the version in the future?

Reply:
I don't believe so, unless something significantly changes with the ubuntu_containerd node os images causing the required kernel modules to be not available.